lib/api/core/models/repositories/userRepository.js

```diff
@@ -190,6 +198,8 @@ class UserRepository extends Repository {
     return super.delete(user, options)
       .then(() => this.kuzzle.repositories.token.deleteByUserId(user._id))
       .then(() => removeUserStrategies(this.kuzzle, user))
+      // @todo test this
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)